src/humanloop/client.py

```diff
@@ -78,11 +90,124 @@ def __init__(
             follow_redirects=follow_redirects,
             httpx_client=httpx_client,
         )
+
+        self._tracer_provider = TracerProvider(
+            resource=Resource(
+                attributes={
+                    "instrumentor": "humanloop.sdk",
+                }
+            ),
+        )
+        instrument_provider(provider=self._tracer_provider)
+        self._tracer_provider.add_span_processor(
+            HumanloopSpanProcessor(
+                exporter=HumanloopSpanExporter(
+                    client=self,
+                )
+            ),
+        )
+        tracer = self._tracer_provider.get_tracer("humanloop.sdk")
+        set_tracer(tracer)
+
         eval_client = ExtendedEvalsClient(client_wrapper=self._client_wrapper)
         eval_client.client = self
         self.evaluations = eval_client
         self.prompts = ExtendedPromptsClient(client_wrapper=self._client_wrapper)
 
+    def prompt(
+        self,
+        # TODO: Template can be a list of objects
```

Comment:
Not supporting chat template is a blocker. 

Also what about the other params like response_format? 

This should just mirror the params supported by the Prompt request body in the SDK

Reply:
Regarding this: can you provide an example of how chat template works? I see that the template can be a list of dicts, are those supposed to be multiple system messages, or is it a pre-scripted interaction e.g.
```
1. System: "Hello I am an assistant specialised in {topics}"
2. Assistant: "Hello how can I help?"
3. User: "{question}"
```

Reply:
Can be a single system message, or a pre-scripted set of messages that prime the model with the desired behaviour (including user and assistant messages)

Reply:
ack, now supports list of messages